app/models/invoice.rb

```diff
@@ -0,0 +1,5 @@
+class Invoice < ApplicationRecord
+    belongs_to :user
+    has_many :items
+    validates :invoice_number, :user_id, presence: true
```

Comment:
if you add the belongs_ and foreign key to the migration file, see other comment - then you don't need to check the user_id presence here.

Reply:
But would still keep the belongs_to association here as well as in the migration file?

Reply:
Yes, you still keep the belongs_to association here, you just don't need to validate the presence of a user_id because when you add the belongs_to and foreign Key to the migration, an invoice cannot be created without that association. :)